spec/output_formats/dir_spec.rb

```diff
@@ -1,8 +1,45 @@
 require 'spec_helper'
 require 'ronin/recon/output_formats/dir'
+require 'tmpdir'
 
 describe Ronin::Recon::OutputFormats::Dir do
+  subject { described_class.new(path) }
+
+  let(:path) { Dir.mktmpdir }
```

Comment:
Call `Dir.mktmpdir` with an name argument like `ronin-recon` or `ronin-recon-output-dir`, etc, so if the directory accidentally gets left behind, we know where it came from.